src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #060b35;
+  font-weight: 400;
+  --light-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.content__card {
+  margin: 50px;
+}
+
+.card__img {
+  display: block;
+  width: 100%;
+  height: 134px;
+  margin-bottom: 40px;
+  padding: 0 3px;
+}
+
+.card__title {
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  gap: 5px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-size: none;
+  background-position: center;
+}
+
+.stars__star--active {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__raiting-caption {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  display: block;
+  width: 100%;
+  padding: 12px 0;
+  background: var(--light-blue);
+  border: none;
+  border-radius: 5px;
+  color: #fff;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  cursor: pointer;
+}
```

Comment:
height of button shouldn't change on hover(use box-sizing)

Reply:
done